tools/checkstyle/checkStyle.xml

```diff
@@ -17,11 +17,21 @@
   ~
```

Comment:
change `checkStyle.xml` to `checkstyle.xml` make sense.

Reply:
+1, we can use `checkstyle.xml` as the file name.